core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSample.java

```diff
@@ -751,9 +751,10 @@ public Sample build() {
           return SampleImpl.sampleAnalyte(this);
         } else if (sampleTissue != null) {
           return SampleImpl.sampleTissue(this);
+        } else {
+          return SampleImpl.sampleTissueProcessing(this);
         }
       }
-      throw new IllegalArgumentException("No sample can be built with the specified parameters.");
```

Comment:
SampleTissueProcessing will be a detailed sample that contains only a SampleAdditionalInfo. So this works, though I would prefer to see some positive testable reason that a sample is a SampleTissueProcessing rather than relying on the fact that it doesn't fit in any other category.